cluster/addons/fluentd-gcp/fluentd-gcp-image/fluent.conf

```diff
@@ -70,7 +70,18 @@
 <match reform.**>
   type record_reformer
   enable_ruby true
-  tag kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+  tag raw.kubernetes.${tag_suffix[4].split('-')[0..-2].join('-')}
+</match>
+
+# Detect exceptions in the log output and forward them as one log entry.
+<match raw.kubernetes.**>
+  type detect_exceptions
+  remove_tag_prefix raw
```

Comment:
Unfortunately, this does not work because detect_exceptions is an output plugin that emits new log entries. This means that fluentd will start matching rules from scratch and will enter an endless loop.